### ISSUE TITLE
Badge counter always ON

### ISSUE THREAD

**Issue by ealdraed** [rNeomy/proxy-switcher#40]
Hello,

I installed the WebExtension in a fresh profile of Firefox 57 (64-bit). I was able to disable the badge counter and the setting was saved even after a restart. However, after installing uBlock Origin the badge counter will show on corresponding websites although it is disabled in settings. I experimented and installed Adblock Plus and it will also cause the badge counter to appear. This even happens when no filter list is activated. Any idea what is happening there?

**Comment by rNeomy** [comment]
I am not able to reproduce this. So when uBlock is installed along with the extension, although it is not supposed to show the number of broken networks requests for tabs, still it does show?

**Comment by ealdraed** [comment]
> I am not able to reproduce this. So when uBlock is installed along with the extension, although it is not supposed to show the number of broken networks requests for tabs, still it does show?

Correct! And the same happens with Adblock Plus, so I thought it may not be related to uBlock. Disabling uBlock/Adblock and refreshing the site is sufficient to restore the normal behaviour (i.e. with badge counter disabled, no numbers are shown on the icon).

I also tried Firefox Portable with a fresh profile -> same behaviour.

**Comment by rNeomy** [comment]
I am still not able to reproduce. See the screen cast
[screen.zip](https://github.com/rNeomy/proxy-switcher/files/1503070/screen.zip)
